pgxn/neon/libpagestore.c

```diff
@@ -909,6 +910,7 @@ call_PQgetCopyData(shardno_t shard_no, char **buffer)
 			neon_shard_log(shard_no, LOG, "no response from pageserver for %0.3f s, disconnecting (socket port=%d)",
 					   INSTR_TIME_GET_DOUBLE(since_start), port);
 			pageserver_disconnect(shard_no);
+			--MyNeonCounters->getpage_stuck_requests_total;
```

Comment:
Prometheus metric counters can never go down, they can only be incremented. That's what `_total` means